### PR TITLE
Remove all references of `co.appbrewery`

### LINKED ISSUE #208: Remove all references of `co.appbrewery`

### PR DESCRIPTION
**Pull request by prestosole**
.

### CODE REVIEW

File: packages/clima_ui/android/app/src/main/kotlin/co/prestosole/clima/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package co.prestosole.clima;
+import io.flutter.embedding.android.FlutterActivity
+
+class MainActivity: FlutterActivity() {
```
Comment (triallax) [inline packages/clima_ui/android/app/src/main/kotlin/co/prestosole/clima/MainActivity.kt]: This file isn't necessary at all (nor is the old Java one); you can change `.MainActivity` in `android/app/src/main/AndroidManifest.xml` to `io.flutter.embedding.android.FlutterActivity` and it should function identically.

**Review by triallax — CHANGES_REQUESTED** [review]

File: packages/clima_ui/android/app/src/main/kotlin/co/prestosole/clima/MainActivity.kt (hunk 1 above)
Comment (prestosole) [reply packages/clima_ui/android/app/src/main/kotlin/co/prestosole/clima/MainActivity.kt]: Should I open a separate PR for this?

Comment (triallax) [reply packages/clima_ui/android/app/src/main/kotlin/co/prestosole/clima/MainActivity.kt]: Certainly possible, but then you'll have to change this file back to Java, so might as well remove it altogether like I said, right?

Comment (prestosole) [reply packages/clima_ui/android/app/src/main/kotlin/co/prestosole/clima/MainActivity.kt]: I guess so

**Review by triallax — APPROVED** [review]

**Comment by triallax** [comment]
Please don't remove the PR template next time; no point if it's not filled. :)

**Pull request merged**